alpha/lib/model/category.php

```diff
@@ -49,6 +56,45 @@ class category extends Basecategory implements IIndexable, IRelatedObject
 	 */
 	protected static $incrementedEntryIds = array();
 	
+	/**
+	 * @return bool
+	 */
+	public function getIsAggregationCategory() {
+		return $this->getFromCustomData(self::IS_AGGREGATION_CATEGORY, null, false);
+	}
+
+	/**
+	 * @return string
+	 */
+	public function getAggregationCategories() {
```

Comment:
It would be cleaner to return an array here instead of a string. If aggregation categories are not set, it should return an empty array and not null